CMakeLists.txt

```diff
@@ -471,6 +471,15 @@ else (ENABLE_FAILPOINTS)
     message (STATUS "Failpoints are disabled")
 endif (ENABLE_FAILPOINTS)
 
+# Since JEMALLOC_NARENAS is introduced for very limited usage, default to -1
+set (JEMALLOC_NARENAS_DEFAULT -1)
+option (JEMALLOC_NARENAS "Set Jemalloc narenas " ${JEMALLOC_NARENAS_DEFAULT})
+if (JEMALLOC_NARENAS LESS_EQUAL 0)
+    message (STATUS "No jemalloc narenas settings")
+else (JEMALLOC_NARENAS LESS_EQUAL 0)
+    message (STATUS "Set jemalloc narenas ${JEMALLOC_NARENAS}")
+endif (JEMALLOC_NARENAS LESS_EQUAL 0)
```

Comment:
Since TiFlash's required version of CMake is v3.21 which is larger than v2.8, the condition in `else` and `elseif` can be omitted. https://cmake.org/cmake/help/v2.8.0/cmake.html#command:if
The other if-else clauses in this file also omit these conditions in `else` and `elseif`. It's better to keep them consistent.

Reply:
Nice suggestions, done